README.md

```diff
@@ -39,6 +39,47 @@ Or to some custom path:
 ```
 make PREFIX=/some/custom/path install
 ```
+## Ubuntu (and probably Debian)
+
+Install nvim, prerequisites, Rust + Cargo.
+
+* Use rustup.rs -- https://rustup.rs
+* This example has to be done as root since the rustup installer installs cargo in one's home dir
+
+Example (Tested on Ubuntu 23.04)
+
+``` shell
+#become root
+sudo su -
+
+#install prereq (see below)
+apt update
+apt install neovim
+apt install libgtk-4-dev -y
+
+#install rust+cargo
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
... precisely because of this 👆🏼. You've never told the user to quit being root and you are now telling them to install foreign software into their distro.